pom.xml

```diff
@@ -227,6 +253,14 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <version>3.2.0</version>
+                    <configuration>
+                        <detectJavaApiLink>false</detectJavaApiLink>
+                    </configuration>
+                </plugin>
```

Comment:
Is mvn javadoc:javadoc passing? If it fails maybe add <failOnError>false<..

Reply:
It fails when you run the "mvn release:perform" step in case you are on JDK14

I cannot reproduce the error with a simple 
`mvn javadoc:javadoc `